app/main.py

```diff
@@ -1 +1,40 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
In the `feed` method of the `Animal` class, you can simplify the code by removing the `else` clause. According to the checklist, you should avoid using an extra `else` statement when it's not needed. Here's how you could rewrite it:

python
if self.is_hungry:
    print(f'Eating {self.appetite} food points...')
    self.is_hungry = False
    return self.appetite
return 0
```